source/rust_verify/src/main.rs

```diff
@@ -107,14 +107,21 @@ pub fn main() {
     let total_time = total_time_1 - total_time_0;
 
     let times_ms_json_data = if verifier.args.time {
+        fn compute_total(
+            verifier: &rust_verify::verifier::Verifier,
+            f: impl Fn(&rust_verify::verifier::BucketStats) -> std::time::Duration,
+        ) -> u128 {
+            verifier.bucket_stats.iter().map(|(_, v)| f(v)).sum::<std::time::Duration>().as_millis()
+        }
+
         let mut smt_init_times = verifier
             .bucket_stats
             .iter()
             .filter(|(k, _)| k.function().is_none())
```

Comment:
Is this filtering that's causing the issue? I admittedly forget why this line is here.
@tjhance @achreto ?

Reply:
> Is this filtering that's causing the issue? I admittedly forget why this line is here.

It looks to me like the filtering is to split out `spinoff_prover` functions when reporting per-module statistics, so that they are reported separately from the modules that contain them.  This sounds useful to me, since `spinoff_prover` functions are often expensive functions that you'd want to keep an eye on individually when profiling.  I suspect, though, that it wasn't intended for these to be excluded from the total times.